### DISCUSSION TITLE
postgres memory

### DISCUSSION THREAD

**Discussion by arberrexhepi** [FlowiseAI/Flowise#961] [Ideas]
Make this truley perfect and add postgres for memory, it will give us the option to not have to opt in to crazy pricing schemes out there!
Also there is no setting for SSL mode on the Vector Store for postgres is there a workaround for now?

edit: It seems to be possible to do this via custom tool with webhooks to make.com but that's a bit "laborious"

**Comment by HenryHengZJ** [comment]
you mean postgres for memory node or vector store node?

**Reply by arberrexhepi** [reply]
For memory node.
As for the SSL Mode that is a mention for the vector store node
